code/datums/gamemode/role/role.dm

```diff
@@ -522,6 +522,67 @@
 	name = BLOBOVERMIND
 	id = BLOBOVERMIND
 	logo_state = "blob-logo"
+	greets = list(GREET_DEFAULT,GREET_CUSTOM)
+	var/countdown = 60
+
+/datum/role/blob_overmind/New(var/datum/mind/M, var/datum/faction/fac=null, var/new_id)
+	..()
+	wikiroute = role_wiki[BLOBOVERMIND]
+
+/datum/role/blob_overmind/OnPostSetup()
```

Comment:
Seems like this does nothing at all?

Reply:
It did at some point then I removed what it did so yeah